desktop/src/main/resources/bisq.css

```diff
@@ -432,6 +466,20 @@
     -fx-background-color: -bisq-content-bg;
 }
 
+/* Primary green button */
+.bisq-primary-button {
```

Comment:
Can we apply that to the `default` pseudo class?
So we dont need to add a style class as in Bisq 1 but apply it to all buttons which are set as default buttons.

Reply:
Ah just saw its not the defult button... For the welcome screen buttons we need a css class as they are larger as normal buttons. So maybe just rename the class to make it more clear. I misinterpreted bisq-primary-button as general css for primary buttons.

Reply:
[Fixed](https://github.com/bisq-network/bisq2/pull/296/commits/b7fb5589cd442125fe59724d87fa30a41b97ac3e)